sentry/src/main/java/io/sentry/util/HintUtils.java

```diff
@@ -16,6 +19,21 @@ public final class HintUtils {
 
   private HintUtils() {}
 
+  public static void setIsFromHybridSdk(@NotNull Hint hint, @NotNull String sdkName) {
+    if (sdkName.startsWith(SENTRY_REACT_NATIVE_SDK_NAME)
+        || sdkName.startsWith(SENTRY_DART_SDK_NAME)) {
+      HintUtils.setIsFromHybridSdk(hint, true);
+    }
+  }
+
+  public static void setIsFromHybridSdk(@NotNull Hint hint, boolean isFromHybridSdk) {
```

Comment:
M: This can be private or inlined with the other `setIsFromHybridSdk` method, I don't see it being called from somewhere else.